src/platform/silabs/ConnectivityManagerImpl_WIFI.cpp

```diff
@@ -220,6 +220,28 @@ void ConnectivityManagerImpl::_OnWiFiStationProvisionChange()
     DeviceLayer::SystemLayer().ScheduleWork(DriveStationState, NULL);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+CHIP_ERROR ConnectivityManagerImpl::_GetSEDIntervalsConfig(ConnectivityManager::SEDIntervalsConfig & SEDIntervalsConfig)
+{
+    // For now Wi-Fi uses DTIM power save mode
+    SEDIntervalsConfig.ActiveIntervalMS = chip::System::Clock::Milliseconds32(50);
```

Comment:
Magic numbers please fix

Reply:
why arent you doing the configurations the same thread does them? and it was does it need to be hardcoded?